### PR TITLE
add 2nd attempt impl. with demo

### PR DESCRIPTION
**Pull request by olmjo**
Clearly much simpler than the first attempt -- was completely missing the obvious approach of writing to articles/ which @jjallaire pointed out.

The only way I can think of to circumvent the copy in the Makefile is at the knitr level and I don't know enough about which knitr functions are doing what (and can't glean it quickly enough).

### CODE REVIEW

File: src/2014-10-14-img.Rmd [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (jjallaire) [inline src/2014-10-14-img.Rmd]: Let's remove this from the PR

File: _scripts/knit.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ knit <- function (inputFile, outputFile) {
   stem <- tools::file_path_sans_ext(inputFile)
   prefix <- paste(stem, "-", sep="")
   knitr::opts_chunk$set(cache.path=file.path('cache', prefix),
-                        fig.path=file.path('figure', prefix))
+                        fig.path=file.path('../figure', prefix))
```
Comment (jjallaire) [inline _scripts/knit.sh:12]: Couldn't this just be '../articles/figure' then you could get rid of the changes to Makefile and _config.yml ?

Comment (jjallaire) [inline _scripts/knit.sh:12]: There's one other problem -- if multiple Rmd files include unnamed chunks then they will overwrite each other's figures. I think you need to do something to derive the fig.path from the basename of the input Rmd.

**Comment by olmjo** [comment]
**multiple unnamed chunks**: I don't see any evidence of that being true, but I might be missing something. The basename of a fig from `YYYY-MM-DD-<slug>.Rmd` is `YYYY-MM-DD-<slug>-<figname>.png` and `<figname>` within a contribution is already handled from knitr. The only conflict I could see would be if there were two contributions on the same day which created `YY-MM-DD-a-b-c.png`, where a-b is the slug from File 1 and c is the chunk name from File 1, but a is the slug from File 2 and b-c is the chunk name from File 2.

**no need to change Makefile and config**: Not without changes inside the knitting. _Just_ changing to `../articles/figure` means that any given HTML file is looking for an image in `/articles/<slug>/../articles/figure/` which obviously doubles `articles`.

**Pull request merged**